doc/src/manual/noteworthy-differences.md

```diff
@@ -121,7 +121,7 @@ For users coming to Julia from R, these are some noteworthy differences:
     of the form `if cond; statement; end`, `cond && statement` and `!cond || statement`. Assignment
     statements in the latter two syntaxes must be explicitly wrapped in parentheses, e.g. `cond && (x = value)`.
   * In Julia, `<-`, `<<-` and `->` are not assignment operators.
-  * Julia's `->` creates an anonymous function, like Python.
+  * Julia's `->` creates an anonymous function, like Haskell.
```

Comment:
In Haskell you need an additional `\` in the beginning, like `\x -> x`. So this is slightly misleading.